### DISCUSSION TITLE
job store with custom detail

### DISCUSSION THREAD

**Discussion by timeworkcoder** [agronholm/apscheduler#663] [Q&A]
Hello,

First of all great library, it is very helpful

however I would like to remove multiple job when problems occur for the data i receive, thus I would like to remove job based on the task id I provide(Which will be duplicate, due to multiple job are come from same request). 

Is there a way to add custom detail to jobstore, so that i could use that detail to remove multiple job with single remove job function?

**Comment by agronholm** [comment]
First of all, which version of the library are you talking about? The terminology has greatly changed in v4.0.

**Reply by timeworkcoder** [reply]
Hi, is 3.9.1
